src/main/java/frc/lib/util/loggingUtil/LogManager.java

```diff
@@ -0,0 +1,173 @@
+package frc.lib.util.loggingUtil;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.function.BooleanSupplier;
+import java.util.function.DoubleSupplier;
+import java.util.function.IntSupplier;
+
+import edu.wpi.first.util.datalog.BooleanLogEntry;
+import edu.wpi.first.util.datalog.DataLog;
+import edu.wpi.first.util.datalog.DoubleArrayLogEntry;
+import edu.wpi.first.util.datalog.DoubleLogEntry;
+import edu.wpi.first.util.datalog.IntegerLogEntry;
+import edu.wpi.first.wpilibj.DataLogManager;
+
+/**
+ * Utilty class for logging data to the DataLogManager.
+ * View logs using MechanicalAdvantage's advantage scope (https://github.com/Mechanical-Advantage/AdvantageScope)
+ */
+public class LogManager {
+
+  private static DataLog log = DataLogManager.getLog();
+
+  // These are array lists of log entry classes from WPI. appending to a log entry automatically adds to the log file.
+  private static ArrayList<DoubleLogEntry> doubleLogs = new ArrayList<>();
+  private static ArrayList<DoubleArrayLogEntry> doubleArrayLogs = new ArrayList<>();
+  private static ArrayList<BooleanLogEntry> booleanLogs = new ArrayList<>();
+  private static ArrayList<IntegerLogEntry> intLogs = new ArrayList<>();
+
+  // These are the suppliers, or functions that return values. This is how the values are accessed.
+  private static ArrayList<DoubleSupplier> doubleValues = new ArrayList<>();
+  private static ArrayList<DoubleSupplier[]> doubleArrayValues = new ArrayList<>();
+  private static ArrayList<BooleanSupplier> booleanValues = new ArrayList<>();
+  private static ArrayList<IntSupplier> intValues = new ArrayList<>();
+
+  // These are the log entries that are not updated periodically, they just receive individual values.
+  private static HashMap<String, DoubleLogEntry> individualDoubleLogs = new HashMap<>();
+  private static HashMap<String, DoubleArrayLogEntry> individualDoubleArrayLogs = new HashMap<>();
+  private static HashMap<String, BooleanLogEntry> individualBooleanLogs = new HashMap<>();
+  private static HashMap<String, IntegerLogEntry> individualIntegerLogs = new HashMap<>();
+
+  
+
+  /**
+   * @deprecated Use {@link #addDouble(String, double)} with a single value instead.
```

Comment:
Are any of these deprecated calls used in the code we are bringing in?  If not, we should remove the deprecated call definitions.